.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+yarnPath: .yarn/releases/yarn-3.1.0.cjs
```

Comment:
why not the current stable 3.2.0 release?

Reply:
done.